msteams-platform/concepts/messaging-extensions/create-extensions.md

```diff
@@ -0,0 +1,258 @@
+---
+title: Action based messaging extensions
+description: Describes how to develop action based messaging extensions
+keywords: teams messaging extensions messaging extensions search
+ms.date: 11/07/18
```

Comment:
Please update the date to be the date we publish.

Reply:
Done, assuming we publish today.